web/backend/app/services/starry_kids/moon.rb

```diff
@@ -64,15 +66,15 @@ def convert_to_emoji(index)
           '🌗'
         when 6
           '🌖'
-        when 7
+        else
           '🌕'
         end
       end
     end
   end
 end
 
-if $0 == __FILE__
+if $PROGRAM_NAME == __FILE__
   moon = StarryKids::Moon
-  p moon.emoji
+  Rails.logger.debug moon.emoji
```

Comment:
```suggestion
  print moon.emoji
```

多分`rubocop -A`したときに勝手に書き換わったっぽい